### PR TITLE
feat: Nostr signing screens 

### LINKED ISSUE #2212: Improve Nostr signing screens
Currently, Nostr signing screens mostly show not very readable data by humans. I propose some changes to displayed info and copy.

In the screens, on the left side of red line there's current implementation and on the right side are proposed improvements. In some cases I propose 2 version on the right side, one which requires less code work, other a bit more but improves UX further. 

None of the changes can rely on Nostr relays 

[Link to Figma file](https://www.figma.com/file/AMzMvcwrr2IdunqMakaB8r/navigation%2C-dropdown%2C-receive-onchain-work%2C-connectors%2C-nostr-events?node-id=40%3A5770&t=Qfd4nheGzTH2es6b-0)

**Sign a note:**
Screenshot 2023-03-14 at 20 50 18

**React to a note:**
Screenshot 2023-03-14 at 20 50 59

**Send encrypted DM:**
Screenshot 2023-03-14 at 20 55 12

**Zap:**
Screenshot 2023-03-14 at 20 59 47

**Update badges:**
Screenshot 2023-03-14 at 20 59 31

**Update followers list:**
Screenshot 2023-03-14 at 21 01 47

**Update profile metadata:**
Screenshot 2023-03-14 at 21 02 19

**Update profile metadata:**
Screenshot 2023-03-14 at 21 02 53

### PR DESCRIPTION
**Pull request by SiddheshKukade**
### Describe the changes you have made in this PR
Currently, Nostr signing screens mostly show not very readable data by humans. I propose some changes to displayed info and copy.
### Link this PR to an issue [optional]

Fixes #2212

### Type of change

(Remove other, not matching type)
 
- `feat`: New feature (non-breaking change which adds functionality)
 

 
#### Working Screenshots:
- Update Followers List:
image

- React to a note:
image

- Update profile metadata:
 image
image
_Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration_

### Checklist

- [x] My code follows the style guidelines of this project and performed a self-review of my own code
- [x] New and existing tests pass locally with my changes
- [x] I checked if I need to make corresponding changes to the documentation (and made those changes if needed)

### CODE REVIEW

**Comment by SiddheshKukade** [comment]
Hello @stackingsaunter, please let me know if the current progress looks good to you.

**Comment by stackingsaunter** [comment]
@SiddheshKukade 

Thanks for contributions! I tested the build. 
Contact list is perfect. Reaction, If we don't show note address, could it be formatted in this way?:
Screenshot 2023-03-31 at 00 24 56

So we don't have two "reaction" next to each other.

**Comment by SiddheshKukade** [comment]
@stackingsaunter Ok, I'll do that

**Comment by SiddheshKukade** [comment]
I was having exams for the past 2 weeks, So I had to take a break. Sorry for the delay here

🥳 more additions to Nostr screens:  
image
image

/ cc
@stackingsaunter 

Feedback would be appreciated.

**Comment by stackingsaunter** [comment]
Hey @SiddheshKukade, sorry for such late review. 

I get this error and blank alby popup while using this build on snort and primal. First two permissions work well, then this appears (there shouldn't be a third one I think?). I cannot do any action because I get this error.

Screenshot 2023-04-25 at 22 21 05
Screenshot 2023-04-25 at 22 22 53

**Comment by SiddheshKukade** [comment]
@stackingsaunter , hello 

sorry for my delayed response I had a small left eye surgery in the past week, so I had to take a break and now I'm feeling better. 
I'll get these things fixed in a few days. 
Thanks for reviewing 👍🏻

File: src/app/components/ContentMessage/index.tsx [hunk 1]
```diff
@@ -1,15 +1,92 @@
+import { t } from "i18next";
+
+/* eslint-disable  @typescript-eslint/no-explicit-any */
 type Props = {
   heading: string;
-  content: string;
+  content:
```
Comment (rolznz) [inline src/app/components/ContentMessage/index.tsx:6]: I believe ContentMessage should be a generic component and the content type should not be modified here

File: src/i18n/locales/en/translation.json [hunk 2]
```diff
@@ -743,11 +743,16 @@
...
       "allow_sign": "Allow {{host}} to sign:",
       "allow_sign_event": "Allow {{host}} to sign a {{kind}}:",
+      "allow_sign_event_contacts": "Allow {{host}} to sign an update to contacts list",
       "view_details": "View details",
       "hide_details": "Hide details",
       "no_content": "(No content)",
       "block_and_ignore": "Block and ignore {{host}}",
       "block_added": "Added {{host}} to the blocklist, please reload the website.",
+      "reactions": {
+        "like": "🤙 Like"
```
Comment (rolznz) [inline src/i18n/locales/en/translation.json:753]: I don't think we need to add a translation for each emoji. The emoji should be self-explanatory

**Comment by rolznz** [comment]
Apologies for the late review. I think the implementation needs to be changed quite significantly. This PR seems to be stale and the original feedback was not addressed. I will close it for now. Please let me know if it is still being worked on.

**Pull request closed without merging**